ecpp

```diff
@@ -65,7 +65,7 @@ else:
             config_buff = pkg_resources.resource_string("ecpp", filename)
             config_buff = str(config_buff, "utf-8")
         else:
-            with open(filename, "r+") as fp:
+            with open(filename, "r") as fp:
```

Comment:
The `r` is default so it might even make sense to skip that argument or even better use `rb` mode to avoid messing up with the line ending if somebody would like to run it with some windows files.

Reply:
`rb` is not what we want, as you can see above we're explicitly converting it to `str()` when using package resources, but yeah, skipping it completely is a good idea